### PR TITLE
AB2-87 control center first version

### PR DESCRIPTION
**Pull request by ravjit-cliqz**

### CODE REVIEW

**Comment by ravjit-cliqz** [comment]
I have to review it myself first, there will be some changes

File: mozilla-release/mobile/android/app/src/main/res/layout/global_trackers_fragment.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (spacifici) [inline mozilla-release/mobile/android/app/src/main/res/layout/global_trackers_fragment.xml]: Copyright notice

Comment (spacifici) [reply mozilla-release/mobile/android/app/src/main/res/layout/global_trackers_fragment.xml]: And please add appropriate prefixes like `ghostery_global_trackers_fragment.xml`

File: mozilla-release/mobile/android/app/src/main/res/layout/list_item_category.xml [hunk 2]
```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?><!--Copyright © Cliqz 2018 -->
```
Comment (spacifici) [inline mozilla-release/mobile/android/app/src/main/res/layout/list_item_category.xml]: Copyright and rename the file to `ghostery_list_item_category.xml`

File: mozilla-release/mobile/android/base/strings.xml.in [hunk 3]
```diff
@@ -678,22 +678,57 @@
...
+  <string name="blocked">&blocked;</string>
+  <string name="tracker_info">&tracker_info;</string>
+  <string name="block_all">&block_all;</string>
+  <string name="unblock_all">&unblock_all;</string>
+  <plurals name="total_trackers_found">
+      <item quantity="one">%d &tracker_found;</item>
+      <item quantity="other">%d &trackers_found;</item>
+  </plurals>
+  <plurals name="total_trackers_blocked">
+    <item quantity="one">%d &tracker_blocked;</item>
```
Comment (spacifici) [inline mozilla-release/mobile/android/base/strings.xml.in]: Should not the `%d` be in the string (i.e `<item quantity="one">&tracker_blocked;</item>` and `<!ENTITY tracker_blocked "%d trackers blocked">`).
BTW, this thing will not compile as soon as we have Polish language support, please add also `two`, `few` and `many`. For the English language just copy the same plural string.

File: mozilla-release/mobile/android/base/java/org/mozilla/gecko/home/HomePager.java [hunk 4]
```diff
@@ -203,6 +206,24 @@ public void onAddPanel(@DrawableRes int resId) {
...
+        addOnLayoutChangeListener(new OnLayoutChangeListener() {
+            @Override
+            public void onLayoutChange(View v, int left, int top, int right, int bottom, int oldLeft, int oldTop, int oldRight, int oldBottom) {
+                Log.d("Layout changed", Integer.toString(top));
+            }
+        });
+    }
+
+    @Override
+    protected void onLayout(boolean changed, int l, int t, int r, int b) {
```
Comment (spacifici) [inline mozilla-release/mobile/android/base/java/org/mozilla/gecko/home/HomePager.java]: Remove this method

File: mozilla-release/mobile/android/base/java/org/mozilla/gecko/home/HomePager.java [hunk 5]
```diff
@@ -20,6 +20,7 @@
 import android.support.v4.content.Loader;
 import android.text.TextUtils;
 import android.util.AttributeSet;
+import android.util.Log;
```
Comment (spacifici) [inline mozilla-release/mobile/android/base/java/org/mozilla/gecko/home/HomePager.java]: Please, do not log.

**Pull request merged**